src/openms/source/CHEMISTRY/ISOTOPEDISTRIBUTION/FineIsotopePatternGenerator.cpp

```diff
@@ -46,7 +46,7 @@ namespace OpenMS
 
     if (use_total_prob_)
     {
-        IsotopeDistribution result(IsoSpecTotalProbWrapper(formula, stop_condition_).run());
+        IsotopeDistribution result(IsoSpecTotalProbWrapper(formula, 1.0-stop_condition_).run());
```

Comment:
this changes results, right?

Reply:
Basically before:

```
FineIsotopePatternGenerator gen(0.01, false, false);
FineIsotopePatternGenerator gen(0.99, false, true);
```

were similar-ish, these two are similar 

```
FineIsotopePatternGenerator gen(0.01, false, false);
FineIsotopePatternGenerator gen(0.01, true, false);
```

so it makes logically more sense: closer to zero = more results. Agreed?

Reply:
Fine with me